### ISSUE TITLE
docs: `.spec.volumeName` is set with name of PV and not PVC

### ISSUE THREAD

**Issue by mloskot** [stackitcloud/rename-pvc#72]
In [README.md](https://github.com/stackitcloud/rename-pvc/blob/436e743e91617681c9e97f0741f698bd9626c5cd/README.md), in section "How does it work?", there is this point:

> 1. Creates the new PVC with the `.spec.volumeName` set to the existing PVC 

which I believe it should read

> 1. Creates the new PVC with the `.spec.volumeName` set to the existing PV

as `.spec.volumeName` takes name of PV, not PVC.

**Comment by dergeberl** [comment]
Yes, you are absolutely right. I have opened a PR to correct it (#73). Thanks for the issue 👍🏻

**Comment by mloskot** [comment]
For a moment I lost confidence in my own understanding - rookie here :) Thanks!
